xserver/playbook.yaml

```diff
@@ -16,12 +16,10 @@
     - name: Creates novnc directory
       file: path=/opt/novnc state=directory
     - name: Download novnc
-      shell: curl -fsSL https://github.com/novnc/noVNC/archive/v0.6.2.tar.gz | tar zxf - -C /opt/novnc && mv /opt/novnc/noVNC-0.6.2 /opt/novnc/noVNC-master
-      when: novnc.stat.exists == False
+      shell: curl -fsSL https://github.com/novnc/noVNC/archive/v1.0.0.tar.gz | tar zxf - -C /opt/novnc && rm -rf /opt/novnc/noVNC-master ; mv /opt/novnc/noVNC-1.0.0 /opt/novnc/noVNC-master
```

Comment:
do we need check stat as was before?

Reply:
no, we need to overwrite if an old version is used